model/project_parser.go

```diff
@@ -685,6 +690,34 @@ func retrieveFile(ctx context.Context, opts GetProjectOpts) ([]byte, error) {
 	}
 }
 
+func retrieveFileForModule(ctx context.Context, opts GetProjectOpts, modules ModuleList, moduleName string) ([]byte, error) {
+	module, err := modules.GetModuleByName(moduleName)
+	if err != nil {
+		return nil, errors.Wrapf(err, "can't get module for module name '%s'", moduleName)
+	}
+	repoOwner, repoName := module.GetRepoOwnerAndName()
+	if opts.Token == "" {
+		conf, err := evergreen.GetConfig()
+		if err != nil {
+			return nil, errors.Wrap(err, "can't get evergreen configuration")
+		}
+		ghToken, err := conf.GetGithubOauthToken()
+		if err != nil {
+			return nil, errors.Wrap(err, "can't get Github OAuth token from configuration")
+		}
+		opts.Token = ghToken
+	}
+	configFile, err := thirdparty.GetGithubFile(ctx, opts.Token, repoOwner, repoName, opts.RemotePath, module.Branch)
+	if err != nil {
+		return nil, errors.Wrapf(err, "error fetching project file for '%s' at '%s'", opts.Ref.Id, opts.Revision)
```

Comment:
this should be different for the module

Reply:
it should print module name and branch now